src/plugins/part.h

```diff
@@ -46,6 +46,7 @@ typedef struct BDPartSpec {
     gchar *uuid;
     gchar *id;
     gchar *type_guid;
+    gchar *type_name;
```

Comment:
Uhm, this looks like ABI break. Suggesting to move this member at the end of the struct.

Reply:
Ok, but it will still be an ABI break even when adding it to the end of the struct, right?

Reply:
Yes, but it's a lesser evil.